src/vs/workbench/contrib/chat/browser/chatAttachmentModel.ts

```diff
@@ -5,10 +5,11 @@
 
 import { Emitter } from '../../../../base/common/event.js';
 import { Disposable } from '../../../../base/common/lifecycle.js';
-import { basename } from '../../../../base/common/resources.js';
-import { URI } from '../../../../base/common/uri.js';
-import { IRange } from '../../../../editor/common/core/range.js';
 import { IChatRequestVariableEntry } from '../common/chatModel.js';
+import { IRange } from '../../../../base/common/range.js';
```

Comment:
This `IRange` import seems to have changed, might be related to the `chatVariables.ts` compile error in CI?